javascript/javascript1/week1/ez-namey.js

```diff
@@ -0,0 +1,5 @@
+let firstWords = ["Easy", "Nice", "Amazing", "Cool", "Fantastic", "Innovative", "Revolutionary", "Futuristic", "AI-driven", "Successful"];
+let secondWords = ["Company", "Firm", "Corporation", "Startup", "Team", "Idea", "Aim", "Job", "Project", "Innovation"];
+const randomNumber = Math.floor(Math.random() * 10);
```

Comment:
What if you need two random numbers, one for every array? 🙂

Reply:
changed to this const startupName = `${firstWords[Math.floor(Math.random() * 10)]} ${secondWords[Math.floor(Math.random() * 10)]}`;